src/app/components/login/login.component.ts

```diff
@@ -55,10 +55,15 @@ export class LoginComponent implements OnInit {
       .pipe(
         catchError((error) => {
           this.loginRequest = false;
+
           if (error instanceof HttpErrorResponse) {
-            this.errorMsg = error.error.errorMessage.Message;
-            if (this.errorMsg.startsWith('Your email address is not confirmed')) {
-              this.resendMsg = true;
+            if (error.status === 0) {
+              this.errorMsg = this.translate.instant('CONNECTIONERROR');
```

Comment:
Tipp: Bevor du jetzt in jeder Komponente alle möglichen Fehler-Responses handeln musst, wäre es vielleicht sinnvoller, allgemeine Fehler wie z. B. den connection error in einen Interceptor auszulagern.

Reply:
Beispiel: https://github.com/TryCatchLearn/DatingApp/blob/master/client/src/app/_interceptors/error.interceptor.ts

Reply:
Ich schreibe dazu einen Jira Task zur Optimierung. Denke das lässt sich schön als eigenes Thema abarbeiten:

https://orso.atlassian.net/browse/ARPA-65